src/main/java/com/rocketden/main/service/RoomService.java

```diff
@@ -272,6 +273,16 @@ public RoomDto updateRoomSettings(String roomId, UpdateSettingsRequest request)
             }
             room.setDuration(request.getDuration());
         }
+
+        // Set new size if not null
+        Integer size = request.getSize();
+        if (size != null) {
+            if (size <= 0 || size > MAX_SIZE + 1 || size < room.getUsers().size()) {
+                throw new ApiException(RoomError.BAD_SETTING);
```

Comment:
nit: maybe add a new custom error specifically for the size field, like "an invalid setting for room size was provided"? I know we currently reuse BAD_SETTING a lot, but I think it might actually be good for us to shift towards more descriptive/specific error messages, especially with all the recent confusing bugs we've come across lol

Reply:
Also, could you fix the typo for `ProblemErrors.INCORRECT_INPUT_COUNT`? One of the TeamsCode members found it while adding a problem lol